### DISCUSSION TITLE
[20396] Failed to execute script chia

### DISCUSSION THREAD

**Discussion by BryanXBY** [swar/Swar-Chia-Plot-Manager#609] [Q&A]
[20396] Failed to execute script chia
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 81, in <module>
  File "chia\cmds\chia.py", line 77, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 135, in create_cmd
  File "chia\plotting\create_plots.py", line 55, in create_plots
ValueError: Length of bytes object not equal to G1Element::SIZE

**Comment by BryanXBY** [comment]
chia_location: C:\Users\bryan\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia.exe


manager:
  check_interval: 60
  log_level: ERROR


log:
  folder_path: D:\Swar-Chia-Plot-Manager\log


view:
  check_interval: 60
  datetime_format: "%Y-%m-%d %H:%M:%S"
  include_seconds_for_phase: false
  include_drive_info: true
  include_cpu: true
  include_ram: true
  include_plot_stats: true


notifications:
  notify_discord: false
  discord_webhook_url: https://discord.com/api/webhooks/0000000000000000/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
  notify_ifttt: false
  ifttt_webhook_url: https://maker.ifttt.com/trigger/{event}/with/key/{api_key}
  notify_sound: false
  song: audio.mp3
  notify_pushover: false
  pushover_user_key: xx
  pushover_api_key: xx
  notify_telegram: false
  telegram_token: xxxxx
  notify_twilio: false
  twilio_account_sid: xxxxx
  twilio_auth_token: xxxxx
  twilio_from_phone: +1234657890
  twilio_to_phone: +1234657890

instrumentation:
  prometheus_enabled: false
  prometheus_port: 9090


progress:
  phase1_line_end: 801
  phase2_line_end: 834
  phase3_line_end: 2474
  phase4_line_end: 2620
  phase1_weight: 33.4
  phase2_weight: 20.43
  phase3_weight: 42.29
  phase4_weight: 3.88


global:
 max_concurrent: 8
  max_for_phase_1: 2
  minimum_minutes_between_jobs: 5


jobs:
    - name: micron
    max_plots: 999
    farmer_public_key: 0xb22222f3bacc28a5e38fb57637c6e10b47c0088ec36e19e65446a9141d154e7913b84dad8e3de948e7eed88a2c6ac7fa
    pool_public_key: 0xb123daebc537e0c1d18819285bdb09fbccb86cec64fd72a46c72f0e61c0a615dfee97c5a6e355fa6a422bac378bf797f
    temporary_directory: J:\
    temporary2_directory:
    destination_directory: K:\out
    size: 32
    bitfield: true
    threads: 8
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 3
    max_concurrent_with_start_early: 7
    initial_delay_minutes: 0
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7 ]

**Comment by zhangyix19** [comment]
Hey, bro!
I've faced the same problem as yours and solved it.
Check if your pool and farmer keys have '0x' at their heads.
If so, delete '0x's and it can work !

**Reply by BryanXBY** [reply]
Yes, there is "0x" at the heads.

**Reply by BryanXBY** [reply]
我万万没有想到是这个问题~~~~我都准备自己用.net core另外写一个P盘的软件了

**Reply by zhangyix19** [reply]
哈哈哈，hpool那个算力签名软件给的是带0x的，也坑了我一晚上
